src/Teams.Business/Services/DiagnosticService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Teams.Data;
+
+namespace Teams.Business.Services
+{
+    public class DiagnosticService : IDiagnosticService
+    {
+        private readonly string pathToConfig;
+        private readonly ApplicationDbContext _dbContext;
+        public DiagnosticService(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext;
+            pathToConfig = "appsettings.json";     
+        }
+
+        public async Task<bool> CheckDbConnection()
+        {
+            try
+            {
+                if (await _dbContext.Database.CanConnectAsync())
+                    return true;
+                else
+                    return false;
+            }
+            catch
+            {
+                return false;
```

Comment:
is it correct behavior?
if db not exists and you try to use CanConnectAsync, will it be failed?
If not here we just miss exception

Reply:
Yes, if Db isn't exist and we delete "Catch", we will get unhandled exception when we call "CanConnect()".